src/options.jl

```diff
@@ -8,6 +8,7 @@ Base.@kwdef struct Options
     import_to_using::Bool = false
     pipe_to_function_call::Bool = false
     short_to_long_function_def::Bool = false
+    long_to_short_function_def::Bool = false
```

Comment:
Sorry for missing that indeed, fixed.

Reply:
no worries we've just started enforcing these recently since v2 will default to MinimalStyle instead of DefaultStyle and so the base options will be different. This way if  you are using another style nothing weird will happen